### ISSUE TITLE
Do not add infinite bounds

### ISSUE THREAD

**Issue by odow** [jump-dev/JuMP.jl#2634]
In this example:
```Julia
using JuMP, SCS
m = Model(with_optimizer(SCS.Optimizer))
@variable(m, -Inf <= x <= Inf)
@objective(m, Max, x)
@constraint(m, x <= 10)
optimize!(m)
```
The bridges should not pass `x - Inf in Nonpositives()` and `x + Inf in Nonnegatives()` to the solver.

x-ref: https://github.com/jump-dev/SCS.jl/pull/205
x-ref: https://github.com/jump-dev/SCS.jl/issues/170

**Comment by blegat** [comment]
> Fixing this in MOI is a much harder task.

I would also argue that the fault is with JuMP here, `MOI.LessThan` and `MOI.GreaterThan` should not contain infinite values. So the best thing the bridge could do is to throw an error. But maybe the error should be thrown in the constructor of `MOI.LessThan` and `MOI.GreaterThan` then.

**Comment by odow** [comment]
Closed by #2618
